app/uk/gov/hmrc/ngrdashboardfrontend/controllers/DashboardController.scala

```diff
@@ -30,45 +30,27 @@ import uk.gov.hmrc.ngrdashboardfrontend.views.html.DashboardView
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class DashboardController @Inject()(
                                      dashboardView: DashboardView,
                                      authenticate: AuthRetrievals,
                                      isRegisteredCheck: RegistrationAction,
                                      ngrConnector: NGRConnector,
-                                     ngrNotifyConnector: NGRNotifyConnector,
                                      mcc: MessagesControllerComponents
                                    )(implicit ec: ExecutionContext, appConfig: AppConfig)
   extends FrontendController(mcc) with I18nSupport {
 
   def show(): Action[AnyContent] = (authenticate andThen isRegisteredCheck).async { implicit request =>
-    ngrConnector.linkedPropertyStatus(CredId(request.credId.getOrElse("")), request.nino).flatMap {
-      case Some(vmvPropertyStatus) =>
-        val cards: Seq[Card] = DashboardHelper.getDashboardCards(true, vmvPropertyStatus.status)
-        val name = request.name.flatMap(_.name).getOrElse(throw new RuntimeException("Name not found"))
-        Future.successful(Ok(dashboardView(
-          cards = cards,
-          name = name,
-          navigationBarContent = createHomeNavBar)))
-      case None => ngrNotifyConnector.getRatepayerStatus(CredId(request.credId.getOrElse(""))).map { vmvPropertyStatus =>
-        val status = vmvPropertyStatus match {
-          case Some(status) if status.activePropertyLinkCount > 0 => Approved
-          case None => Rejected
-          case _ => Rejected
-        }
-
-        val cards: Seq[Card] = DashboardHelper.getDashboardCards(vmvPropertyStatus.isDefined, status)
-        val name = request.name.flatMap(_.name).getOrElse(throw new RuntimeException("Name not found"))
-        Ok(dashboardView(
-          cards = cards,
-          name = name,
-          navigationBarContent = createHomeNavBar))
-
-      }
+    ngrConnector.linkedPropertyStatus(CredId(request.credId.getOrElse("")), request.nino).map { vmvPropertyStatus =>
```

Comment:
There was a ticket to look into request.credId.getOrElse(""), can you please mention the ticket that we are gloing to fix the getOrElse